qcodes/instrument_drivers/Keysight/Infiniium.py

```diff
@@ -152,6 +150,133 @@ def get(self):
         return channel_data
 
 
+class MeasurementSubsystem(InstrumentChannel):
+    """
+    Submodule containing the measurement subsystem commands and associated
+    parameters
+    """
+    # note: this is not really a channel, but InstrumentChannel does everything
+    # a 'Submodule' class should do
```

Comment:
We had a discussion in a bug report about renaming `InstrumentChannel` for this reason (can't find the discussion right now though). Perhaps we should revisit?

Reply:
this can be very easily solved by renaming `InstrumentChannel` class to SubModule` (or `SubInstrument`, whatever) and adding `InstrumentChannel = SubModule` after `SubModule` class definition.

`SubModule` seems like a good name. But what "should this class do"? Contain a piece of functionality with tight reference/coupling to the instrument that it is a part of?

Reply:
I like the name `SubModule` too. I think `InstrumentChannel` already **is** that class. If you guys would like to discuss this more, then let us perhaps open a separate issue.